packer-scripts/remove-default-users

```diff
@@ -22,10 +22,7 @@ __remove_user() {
     return
   fi
 
-  ps -Fwww -u "${u}" || echo "Suppressing ps exit $?"
-  killall -TERM -w -v -u "${u}" || echo "Suppressing killall exit $?"
```

Comment:
Ah, good point. I guess the processes will be shut down at the end of the packer build anyways.